float64 x
float64 y
float64 z
string say
float64 ddd

